ui/admin/app/components/worker-diagram/index.hbs

```diff
@@ -1,6 +1,15 @@
-{{yield
-  (hash
-    SingleFilter=(component 'worker-diagram/single-filter')
-    DualFilter=(component 'worker-diagram/dual-filter')
-  )
-}}
\ No newline at end of file
+{{#if (feature-flag 'target-worker-filters-v2-hcp')}}
+  <WorkerDiagram::DualFilter::Hcp
+    @egressWorkerFilterEnabled={{@egressWorkerFilterEnabled}}
+    @ingressWorkerFilterEnabled={{@ingressWorkerFilterEnabled}}
+  />
+{{else if (feature-flag 'target-worker-filters-v2-ingress')}}
+  <WorkerDiagram::DualFilter
+    @egressWorkerFilterEnabled={{@egressWorkerFilterEnabled}}
+    @ingressWorkerFilterEnabled={{@ingressWorkerFilterEnabled}}
+  />
+{{else}}
+  <WorkerDiagram::SingleFilter
+    @egressWorkerFilterEnabled={{@egressWorkerFilterEnabled}}
+  />
+{{/if}}
```

Comment:
Minor readability update so that it's clear v2-ingress and v2-hcp can both be on at the same time.  The previous conditional could be interpreted to mean they are mutually exclusive.
```suggestion
{{#if (feature-flag 'target-worker-filters-v2-ingress')}}
  {{#if (feature-flag 'target-worker-filters-v2-hcp')}}
    <WorkerDiagram::DualFilter::Hcp
      @egressWorkerFilterEnabled={{@egressWorkerFilterEnabled}}
      @ingressWorkerFilterEnabled={{@ingressWorkerFilterEnabled}}
    />
  {{else}}
    <WorkerDiagram::DualFilter
      @egressWorkerFilterEnabled={{@egressWorkerFilterEnabled}}
      @ingressWorkerFilterEnabled={{@ingressWorkerFilterEnabled}}
    />
  {{/if}}
{{else}}
  <WorkerDiagram::SingleFilter
    @egressWorkerFilterEnabled={{@egressWorkerFilterEnabled}}
  />
{{/if}}
```

Reply:
Updated!